OracleWebLogic/samples/12213-domain-home-in-image/Dockerfile

```diff
@@ -17,7 +17,7 @@
 #
 # Pull base image
 # ---------------
-FROM oracle/weblogic:12.2.1.3-developer
+FROM wlsldi-v2.docker.oraclecorp.com/weblogic:19.1.0.0
```

Comment:
We cannot use 19c image because it has not been released.

Reply:
Carolyne 19.1 is not GA until next year you have to extend the 12.2.1.3 image

Reply:
Also, this is leaking internal Oracle hostnames and versioning information, which is confidential information. In future, please take a little more care before submitted a PR publicly that contains this sort of data.